driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpClientChannel.java

```diff
@@ -23,7 +23,7 @@
 import org.kaazing.k3po.driver.internal.netty.bootstrap.channel.AbstractChannel;
 import org.kaazing.k3po.driver.internal.netty.channel.ChannelAddress;
 
-public class HttpClientChannel extends AbstractChannel<HttpChannelConfig> {
+public class HttpClientChannel extends AbstractChannel<DefaultHttpChannelConfig> {
```

Comment:
I did this to avoid having to cast the config to DefaultHttpChannelConfig, which I think is worse. But I have now found a better solution by creating our own extended version of the ChannelConfig interface.